src/WrapperApp/WrapperApp.tsx

```diff
@@ -52,15 +52,8 @@ function WrapperApp() {
 	}, [isAuthorized, tabsValue]);
 
 	useEffect(() => {
-		const tabTitles = {
-			login: 'Login',
-			editor: 'Editor',
-			inputFiles: 'Input files',
-			simulations: 'Simulations',
-			results: 'Results',
-			about: 'About'
-		};
-		document.title = `${tabTitles[tabsValue as keyof typeof tabTitles] || 'Yaptide'}`;
+		const result = tabsValue.replace(/([A-Z])/g, ' $1');
```

Comment:
Add a comment explaining what this code is doing, ideally providing an example. Something like: "This codes appends a space as a suffix to the string. If `tabsValue` is equal to `Input files` then `result` would be ` Input file`"